lib/Service/LabelService.php

```diff
@@ -91,6 +91,18 @@ public function create($title, $color, $boardId) {
 		}
 
 		$this->permissionService->checkPermission(null, $boardId, Acl::PERMISSION_MANAGE);
+
+		//$this->checkDuplicateTitle($boardId, $title);
```

Comment:
Could you still move this to a separate method. Then we don't have duplicate code.